src/Microsoft.DotNet.Interactive.Tests/KernelCommandNestingTests.KernelCommandResult_Events.cs

```diff
@@ -209,11 +209,11 @@ public async Task Commands_sent_within_the_code_of_another_command_publish_Stand
             returnedValueFromCSharp
                 .Should().BeOfType<List<KernelEvent>>()
                 .Which
-                .Should().ContainSingle<StandardOutputValueProduced>()
```

Comment:
The test is specifically about making sure that `StandardOutputValueProduced` is propagated correctly when one kernel is executing within another's context.

Maybe change the return value on line 200 to a `string` so that the resulting event will still be `StandardOutputValueProduced`?

Reply:
updated it too, pls have a look if that works.